app/scripts/controllers/eventRegistrations.js

```diff
@@ -93,6 +93,50 @@ angular.module('confRegistrationWebApp')
       $scope.queryParameters.block = visibleBlocks;
     };
 
+    // Set initial Hide Column values
+    $scope.columns = ['Started', 'Completed', 'Email'];
+    $scope.columnChecked = [true, true, true];
+    $scope.showStarted = true;
+    $scope.showCompleted = true;
+    $scope.showEmail = true;
```

Comment:
If you end up keeping this logic separate, I think you could keep it all in a single object such as:
```
var builtInColumnsVisible = {
    Email: true,
    Started: true,
    Completed: true
};
```

You could loop through the keys to get the column names (that would probably not be as useful when you have the custom columns in between) and look up the visible state by key in both the dropdown and in the table.